test/unit/uk/gov/hmrc/bindingtariffclassification/scheduler/SchedulerTest.scala

```diff
@@ -119,7 +119,7 @@ class SchedulerTest extends UnitSpec with MockitoSugar with BeforeAndAfterEach w
       given(util.closestRun(job.firstRunTime, job.interval)) willReturn "2018-12-25T12:00:00"
 
       // When
-      new Scheduler(actorSystem, config, schedulerRepository, util, job)
+      createScheduler
```

Comment:
I think it would read better if we changed `createScheduler` to something like `whenTheSchedulerStarts`